### DISCUSSION TITLE
Not able to download Autogen in .NET

### DISCUSSION THREAD

**Discussion by teeshajain17** [microsoft/autogen#2710] [Q&A]
Hi,

I'm new to learning Autogen and work exclusively with .NET. I tried downloading the Autogen NuGet package in Visual Studio 2022, but it keeps asking for my Microsoft or GitHub credentials. I've tried both but still can't download it. I've followed all the steps outlined [here](https://microsoft.github.io/autogen-for-net/articles/Installation.html).

Can anyone help, please?
![Screenshot 2024-05-17 180517](https://github.com/microsoft/autogen/assets/87597536/70346a13-172c-4168-af8f-b35eef36695f)

**Comment by LittleLittleCloud** [comment]
That source is for internal only. If you don’t need to consume nightly build you can simply download packages from nuget.

**Reply by teeshajain17** [reply]
![Screenshot 2024-05-17 230105](https://github.com/microsoft/autogen/assets/87597536/a7a471ad-c527-496b-b8b0-99d6a083826d)
Hey. I tried downloading AutoGen Nuget package from Nuget Package manager only. But it asked me credentials for Microsoft account and when I entered then it shows  me this error

**Reply by LittleLittleCloud** [reply]
Can you run the following command and share the output? It helps me understand what nuget source do you have

```
dotnet nuget list source
```

**Reply by teeshajain17** [reply]
Hey, I checked I had some extra sources available when I was trying the night build. When I removed them and tried again, it worked. Thanks a ton.
